modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm

```diff
@@ -288,8 +288,8 @@ GLOBAL_LIST_EMPTY(customizable_races)
 /datum/species/vampire
 	mutant_bodyparts = list()
 
-/datum/species/hemophage
-	mutant_bodyparts = list()
+/* /datum/species/hemophage
+	mutant_bodyparts = list() */ // Expanded in another file.
```

Comment:
I'm not sure why you're commenting this here, might as well remove it if it's not needed here?